project/Dependencies.scala

```diff
@@ -124,7 +126,8 @@ object Dependencies {
 
   lazy val httpTestkit = l ++= Seq(
     Test.junit, Test.junitIntf, Compile.junit % "provided",
-    Test.scalatest.value.copy(configurations = Some("provided; test"))
+    Test.scalatest.value.copy(configurations = Some("provided; test")),
+    "org.specs2" %% "specs2-core" % specs2Version.value % "provided"
```

Comment:
It would be nice to have it declared in the `Test` section with license info for completeness.